R/utils_render_html.R

```diff
@@ -1023,6 +1025,22 @@ create_source_notes_component_h <- function(data) {
     n_cols <- n_data_cols
   }
 
+  # Get the style attrs for the source notes
+  if ("source_notes" %in% styles_tbl$locname) {
+
+    source_notes_style <- dplyr::filter(styles_tbl, locname == "source_notes")
+
+    source_notes_styles <-
+      if (nrow(source_notes_style) > 0) {
+        source_notes_style$html_style
```

Comment:
This should be `paste(collapse = " ", source_notes_style$html_style)`

Reply:
Done.